### PR TITLE
Add Neo4j quickstart.

### PR DESCRIPTION
**Pull request by michael-simons**
This is the guide for https://github.com/quarkusio/quarkus/pull/3171.

### CODE REVIEW

File: neo4j-quickstart/blah/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*
+!target/*-runner
+!target/*-runner.jar
+!target/lib/*
```
Comment (gsmet) [inline neo4j-quickstart/blah/.dockerignore]: Hmmm, the files are in a `blah` subdirectory. That's really not something we would want :).

Comment (michael-simons) [reply neo4j-quickstart/blah/.dockerignore]: 🤦‍♂️ Moved the stuff too often. Embarrassing.

**Comment by michael-simons** [comment]
That's fixed. I'll take of the guide tomorrow morning.

**Comment by gsmet** [comment]
Merged, thanks!

**Comment by rsvoboda** [comment]
Any tweet to ❤️ ? 

Or need to wait for the next release?

**Comment by michael-simons** [comment]
I waited to see if it’s featured in our weekly “Twin4j” newsletter.  I think it will be in one of the next issues. In the meantime:

https://twitter.com/rotnroll666/status/1150837536543137794?s=21

**Pull request merged**